scripts/ci/gitlab/pipeline/check.yml

```diff
@@ -35,6 +35,16 @@ test-dependency-rules:
   script:
     - ./scripts/ci/gitlab/ensure-deps.sh
 
+test-rust-features:
+  stage:                           check
+  extends:
+    - .kubernetes-env
+    - .test-refs-no-trigger-prs-only
+  variables:
+    CI_IMAGE:                      "paritytech/tools:latest"
```

Comment:
```suggestion
    CI_IMAGE:                      "paritytech/ci-linux:production"
```

Reply:
And actually this section can be removed:
```
  variables:
    CI_IMAGE:                      "paritytech/tools:latest"
```

Reply:
Ahh, okay thanks. Will try.